README.md

```diff
@@ -153,7 +155,9 @@ This file and all other files in this repository are licensed under the
 Apache License, v 2.0 except as noted otherwise in the [LICENSE] file.
 
 [Tutorials on Data Attribute Recommendation]: https://developers.sap.com/mission.cp-aibus-data-attribute.html
+[SDK tutorials]: https://developers.sap.com/group.cp-aibus-data-attribute-sdk.html
 [SAP AI Business Services]: https://help.sap.com/viewer/product/SAP_AI_BUS/SHIP/en-US
+[SDK Setup Tutorial]: https://developers.sap.com/tutorials/cp-aibus-dar-sdk-setup.html
```

Comment:
This is quite a comprehensive setup tutorial. Really nice! 
It looks like they use `virtualenv`. You might need to make sure the workshop does not use `anaconda` if you're using this, to avoid confusion.

Reply:
Thanks! I will keep that in mind. This was all Tobias' work, btw!

Reply:
Yep, pretty cool. Should we ask him to add an `anaconda` setup as well? It might be useful if you decided to use jupyter notebooks for the tutorial.